joshuaCarpentier (copy)/assignments/slidingwindow/lc209/lc209BinaryTree..js

```diff
@@ -0,0 +1,55 @@
+var minSubArrayLen = function(target, nums) {
```

Comment:
It doesn't seem like this solution passes the second test case. Did you test your code before submitting?

Reply:
I was just testing and observing this example I found it my attempt was on the solution without the binary search